mod/05_email/set/type/email_template.rb

```diff
@@ -44,7 +44,7 @@ def email_config args={}
   
   [:subject, :text_message].each do |field_name|
     config[field_name] = process_email_field( field_name, args ) do |field_card|
-      field_card.contextual_content context_card, {:format=>'email_text'}, args
+      field_card.contextual_content context_card, {:format=>'email_text'}, args.merge(:content_opts=>{ :chunk_list=>:inclusion_and_link})
```

Comment:
That seems a bit better, but I don't think changing the chunks processed is the best way to handle this for the url case.  If the [link] syntax is what is throwing off the spam processing, then I think that should be disabled through render options rather than by changing the list of chunk types.  The point is that except for the odd bare hostname case, url processing in the text format is a noop.  Presumably the 'address' case can use a special format class where the link format is overriden?

This is really not a deep technical issue with performance implications or anything, but I think it is important enough to make a collective decision about which approach is better and to be preferred.  Making a couple of Format subclasses with the special link handling is probably the cleanest solution.  Overriding the chunk lists isn't so bad, but I don't like it as much.  I'm not liking the idea of using render args as much as I think about it, and I think Ethan may have a stronger prohibition for that option.

The content specs test file is a good place to add more cases for this.  There are example tests that use non-default formatters (otherwise it always does rich html).